files/build_templates/init_cfg.json.j2

```diff
@@ -135,6 +135,18 @@
             "special_class": "true"
         }
     },
+    "SSH_SERVER": {
```

Comment:
Please don't define defaults here. Previously it was discussed and removed. Refer the link below. Please define your defaults in the 

``` https://github.com/sonic-net/sonic-buildimage/pull/14513#discussion_r1246974269 ```